demos/mediapipe/holistic_tracking/mediapipe_holistic_tracking.py

```diff
@@ -0,0 +1,197 @@
+    print(command)
+    if subprocess.call(command.split()) != 0:
+      sys.exit(-1)
+
+def convert_pose():
+    print("Converting pose detection model")
+    run_command("wget https://github.com/PINTO0309/tflite2tensorflow/raw/main/schema/schema.fbs")
+    run_command("git clone -b v2.0.8 https://github.com/google/flatbuffers.git")
+    run_command("bash build_dependencies.sh")
+    run_command("cp -r  ovms/pose_detection/1/pose_detection.tflite .")
+    run_command("tflite2tensorflow --model_path pose_detection.tflite --flatc_path flatbuffers/build/flatc --schema_path schema.fbs --output_pb")
+    run_command("tflite2tensorflow --model_path pose_detection.tflite --flatc_path flatbuffers/build/flatc --schema_path schema.fbs --output_no_quant_float32_tflite   --output_dynamic_range_quant_tflite   --output_weight_quant_tflite   --output_float16_quant_tflite   --output_integer_quant_tflite")
+    run_command("cp -rf saved_model/model_float32.tflite ovms/pose_detection/1/pose_detection.tflite")
+    run_command("rm -rf pose_detection.tflite")
+    run_command("rm -rf saved_model")
+
+def download_model(model_path: str):
+    """Downloads the oss model from Google Cloud Storage if it doesn't exist in the package."""
+    model_url = _GCS_URL_PREFIX + model_path.split('/')[-1]
+    dst = os.path.join("ovms/", model_path.replace("/","/1/"))
+    dst_dir = os.path.dirname(model_path)
+
+    # Workaround to copy every model in separate directory
+    model_name = os.path.basename(model_path).replace(".tflite","")
+    dir_name = os.path.basename(dst_dir)
+    if dir_name != model_name:
+        dst = dst.replace(dir_name + "/", model_name + "/")
+
+    dst_dir = os.path.dirname(dst)
+    if not os.path.exists(dst_dir):
+        os.makedirs(dst_dir)
+
+    dst_file = os.path.join(dst_dir, os.path.basename(model_path))
+    print('Downloading model to ' + dst_file)
+    with urllib.request.urlopen(model_url) as response, open(dst_file,
+                                                           'wb') as out_file:
+        if response.code != 200:
+            raise ConnectionError('Cannot download ' + model_path +
+                                    ' from Google Cloud Storage.')
+        shutil.copyfileobj(response, out_file)
+
+def prepare_models():    
+    external_files = [
+       # Using short range
+       # 'face_detection/face_detection_full_range_sparse.tflite',
+        'face_detection/face_detection_short_range.tflite',
+        'face_landmark/face_landmark.tflite',
+       # Model loading error
+       # 'face_landmark/face_landmark_with_attention.tflite',
+        'hand_landmark/hand_landmark_full.tflite',
+       # Using full
+       # 'hand_landmark/hand_landmark_lite.tflite',
+        'holistic_landmark/hand_recrop.tflite',
+        'iris_landmark/iris_landmark.tflite',
+        'palm_detection/palm_detection_full.tflite',
+       # Using full
+       # 'palm_detection/palm_detection_lite.tflite',
+       # Need to use OV version

```

Comment:
```suggestion
       # Need to use OV version
```
Here If I understood correctly we converted tflite model to TF and then back to tflite and we still use tflite model and not OV IR representation?

Reply:
Yes